extensions/amp-ad/0.1/amp-ad.js

```diff
@@ -62,6 +65,8 @@ export class AmpAd extends AMP.BaseElement {
         `Unknown ad type "${type}"`
       );
 
+      dev().expectedError(TAG, `ad type "${type}"`);
```

Comment:
Would you mind adding a TODO linking to a tracking issue for this, so that later on we can come back and remove it when the data collection is over?

Reply:
Done

Reply:
Is there a tracking issue # we can include in the TODO? Will make it easier to trace/make sure it isn't forgotten

Reply:
Also done.